v1/lib/v1/searchable/query.rb

```diff
@@ -84,26 +92,55 @@ def self.protect_metacharacters(string)
       def self.string_queries(resource, params)
         query_strings = []
 
+        # exact_field_match: Whether to match exactly with the not_analyzed
+        # index for a field, vs. doing a tokenized search for parts of the
+        # string.  E.g. "University of Pennsylvania" -- search for exactly
+        # that string, or for ("University" OR "Pennsylvaina")?
+        exact_field_match_param = params['exact_field_match'].to_s == 'true'
+
         params.each do |name, value|
           # Skip all query types that are handled elsewhere
           next if value.to_s == ''
           next if name =~ /^.+\.(before|after)$/
 
           if name == 'q'
             fields = field_boost_for_all(resource) + ['_all']
+            # The `q' parameter always wants a tokenized search, so we won't
+            # ask for it to be quoted below in our call to
+            # `.protect_metacharacters'.
+            exact_field_match = false
```

Comment:
@markbreedlove if a `q` term includes quotation marks, such as `"memorial day" parade`, will the phrase "memorial day" still be recognized as a phrase?  Or will the words "memorial" and "day" be tokenized?